packages/client/components/NullCard/NullCard.tsx

```diff
@@ -16,6 +16,12 @@ const AddingHint = styled('div')({
   textAlign: 'center'
 })
 
+const HeadName = styled('div')({
```

Comment:
+1 Let's make naming consistent with other elements, as there is no Head. For example just `<Name>`